axiom/nr_txn.c

```diff
@@ -2479,6 +2536,13 @@ nr_analytics_event_t* nr_error_to_event(const nrtxn_t* txn) {
   nro_set_hash_string(params, "error.class", nr_error_get_klass(txn->error));
   nro_set_hash_string(params, "error.message",
                       nr_error_get_message(txn->error));
+  if (nr_error_get_file(txn->error) && nr_error_get_context(txn->error)) {
```

Comment:
Is it ever possible we'd only one to write a subset of these 4 or do they either all exist or not exist?